### PR TITLE
fixed explodableMeshbits

### PR DESCRIPTION
**Pull request by Nemoel-Tomo**
fixed explodableMeshbits
fixed that undead enemies could get damage

### CODE REVIEW

File: TombEngine/Game/control/los.cpp [hunk 1]
```diff
@@ -133,34 +133,26 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 
 					if (item->ObjectNumber < ID_SHOOT_SWITCH1 || item->ObjectNumber > ID_SHOOT_SWITCH4)
 					{
-						if ((Objects[item->ObjectNumber].explodableMeshbits & ShatterItem.bit) &&
```
Comment (Lwmte) [inline TombEngine/Game/control/los.cpp:136]: Why did you remove these brackets? This potentially may change precedence of operations.

Comment (Nemoel-Tomo) [reply TombEngine/Game/control/los.cpp:136]: added brackets and tested again. It works also with the brackets.

File: TombEngine/Game/control/los.cpp [hunk 2]
```diff
@@ -136,31 +136,23 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 						if ((Objects[item->ObjectNumber].explodableMeshbits & ShatterItem.bit) &&
 							LaserSight)
 						{
-							//if (!Objects[item->objectNumber].intelligent)
-							//{
+							if (item->ObjectNumber == ID_GUARD_LASER)
+							{
+								float angle = phd_atan(LaraItem->Pose.Position.z - item->Pose.Position.z, LaraItem->Pose.Position.x - item->Pose.Position.x) - item->Pose.Orientation.y;
```
Comment (Sezzary) [inline TombEngine/Game/control/los.cpp:141]: Type should probably be `short`, not `float`.

Comment (Nemoel-Tomo) [reply TombEngine/Game/control/los.cpp]: fixed

**Review by Lwmte — APPROVED** [review]
